src/frontend/src/test-e2e/views.ts

```diff
@@ -168,13 +168,17 @@ export class PinAuthView extends View {
       await input.setValue(digit);
     }
   }
+
+  async waitForError(): Promise<void> {
+    await this.browser
+      .$('[data-haserror="true"]')
+      .waitForDisplayed({ timeout: 5_000 });
+  }
 }
 
 export class RecoveryMethodSelectorView extends View {
   async waitForDisplay(): Promise<void> {
-    await this.browser
-      .$('[data-action="cancel"]')
-      .waitForDisplayed({ timeout: 10_000 });
+    await this.browser.$('[data-page="add-recovery-phrase"]').waitForExist();
```

Comment:
This needed to change, because the new test exposed that quite a few pages do have a cancel button. 😉

Reply:
Ditto to use a class variable `RecoveryMethodSelectorView.selector`?